R/class-plotdata-beeswarm.R

```diff
@@ -27,10 +27,14 @@ newBeeswarmPD <- function(.dt = data.table::data.table(),
                          jitter = NULL,
                          median = logical(),
                          evilMode = logical(),
-                         listVarDetails = list('inferredVariable' = NULL,
+                         collectionVariableDetails = list('inferredVariable' = NULL,
                                                'inferredVarPlotRef' = NULL,
-                                               'listVarPlotRef' = NULL,
-                                               'listVarDisplayLabel' = NULL),
+                                               'collectionVariablePlotRef' = NULL,
+                                               'collectionVarDisplayLabel' = NULL),
```

Comment:
my only question is about the display label here. maybe i just missed it, but im not sure i saw it used anywhere. plus it probably makes the most sense for the client to find display labels given variable specs..

Reply:
Sounds good. collectionVarDisplayLabel removed!